romancal/resample/tests/test_resample.py

```diff
@@ -279,6 +279,13 @@ def multiple_exposures(exposure_1, exposure_2):
     return exposure_1
 
 
+def get_resampled_wcs_pixel_scale(wcs):
+    t = wcs.forward_transform
+    for p in t.param_names:
+        if p.startswith("factor"):
```

Comment:
One worries a bit about this breaking in the future.  I don't immediately expect other bits that would go very early in the WCS and would have names that start with factor, but it's worth at least thinking about whether alternatives exist?